packages/deb/hazelcast/DEBIAN/postinst

```diff
@@ -1,5 +1,15 @@
 #!/bin/sh
 echo 'hazelcastDownloadId=CLI_DEB' > "/usr/lib/hazelcast/hazelcast-4.1-BETA-1/bin/download/hazelcast-download.properties"
 
+for filename in /usr/lib/hazelcast/hazelcast-4.1-BETA-1/bin/hz*; do
+  case "$filename" in
+    *bat)
+      ;;
+    *)
+      ln -s "$filename" /usr/bin/"$(basename "$filename")"
```

Comment:
Do we need to remove this in some post uninstall script?

Reply:
@frant-hartm added symlink removal in post uninstall phases, can you please re-review?